lib/notifier/index.js

```diff
@@ -166,6 +166,51 @@ function transactionNotify (tx, rec) {
   })
 }
 
+function complianceNotify (customer, deviceId, ...args) {
+  return Promise.all([
+    settingsLoader.loadLatest(),
+    queries.getMachineName(deviceId)
+  ])
+    .then(([settings, machineName]) => {
+      const notifications = configManager.getGlobalNotifications(settings.config)
+
+      const action = args[0]
+      const period = args[1]
```

Comment:
better to just name the params instead of getting from args

Reply:
Hmm, I don't think I can name the params specifically, as they come from a spread operator. I can deconstruct the array though, instead of selecting elements one by one

Reply:
Hmm, the reason it works is because of the rest parameter.

```js
// these functions should be equivalent
(a, b, c) => {
  return a + b + c
}

(a, ...args) => {
  return a + args[0] + args[1]
}

```